sky/services/rasterizer/rasterizer.mojom

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module rasterizer;
+
+[ServiceName="rasterizer::Rasterizer"]
```

Comment:
I don't think this annotation is needed either because we don't use this interface with ServiceProvider.